README.md

```diff
@@ -46,38 +48,49 @@ git clone https://github.com/elodin-sys/elodin.git
 cd elodin
 ```
 
-### 3. Build and Install Elodin Editor and Elodin DB into your path
+### 3. Enter the Nix Development Shell (this can take awhile to build the first time)
 ```sh
-# this will also build and use the shell below
-just install
+nix develop
 ```
+> [!TIP]
+> The Nix shell supports Oh My Zsh + Powerlevel 10k; for first time configuration run: `p10k configure`
+>
 
-### 4. Enter the Development Shell, run elodin
+### 4. Build and Install Elodin Editor and Elodin DB into your path
 ```sh
-nix develop
-# open elodin, we'll connect to the server we start below
-elodin
+just install
+
+elodin --version
+
+python3 examples/rocket/main.py run
 ```
-> [!TIP]
-> The Nix shell runs Oh My Zsh + Powerlevel 10k, and will run configuration setup on first run if not installed
 
-### 5. Enter the Development Shell, run the server & example simulation
+Open the Elodin editor in a new nix develop shell and connect to the local server
+
+---
+
+## Python SDK Development Setup (Nix-based continued)
+
+### 5. Enter *another* Nix Development Shell, run the server & example simulation
 
 #### Python SDK Development
 ```sh
 # In a new terminal
-nix develop # enter a new nix develop shell
-cd libs/nox-py
-uv venv --python 3.12
-source .venv/bin/activate
+nix develop
+# build the SDK python wheel
+cd libs/nox-py && \
```

Comment:
Does this need to be all one command? I feel like it's a bit clearer/less scary looking the old way but I don't have strong feelings about it.